pkgs/tools/security/gnupg/20.nix

```diff
@@ -3,13 +3,13 @@
 
 # Each of the dependencies below are optional.
 # Gnupg can be built without them at the cost of reduced functionality.
-, pinentry ? null, x11Support ? true
+, pinentry ? null, guiSupport ? true
```

Comment:
The big reason not to is to not break this for people with config overrides set up.